docs/user-guide/migrating-to-2.0.0.md

```diff
@@ -181,7 +181,13 @@ Additionally, if you are using `context.ecmaFeatures` inside of your rules, then
 1. If you're using `context.ecmaFeatures.modules`, rewrite to check that the `sourceType` property of the Program node is `"module"`.
 1. If you're using a non-ES6 feature flag such as `context.ecmaFeatures.jsx`, rewrite to check for `context.parserOptions.ecmaFeatures.jsx`.
 
-If you're not using `ecmaFeatures` in your configuration, then no change is needed.
+If you are using `ecmaFeatures` inside rule unit tests, those need to be rewritten to use `parserOptions`:
```

Comment:
This section basically just repeats what was already said above it. We can probably just say, "If you have a plugin with rules, and you are using RuleTester, then you also need to update the options you pass for rules that use `ecmaFeatures`." Then, show an example.